Source/FactoryGame/Public/FGFoliageTypes.h

```diff
@@ -60,7 +60,7 @@ FORCEINLINE uint32 GetTypeHash( const FFoliageRemovalDataKey& saveDataKey )
 /**
  * A struct that can be used to uniquely identify a foliage instance accross the network. Used for replication. 
  */
-USTRUCT()
+USTRUCT(BlueprintType)
```

Comment:
Avoid modifying base game headers unless absolutely necessary, and when it must be done, leaving a `MODDING EDIT` comment so it can be easily identified. In this case a BlueprintType access transformer should be used.